app/controllers/assessments_controller.rb

```diff
@@ -0,0 +1,69 @@
+class AssessmentsController < ApplicationController
+  before_action :set_contact, only: %i[new create]
+
+  def new
+    @users = User.all
+    @roles = Role.all
+    @type = %w[log schedule].include?(type_param) ? type_param : 'log'
+
+    @need = Need.new
+    @note = Note.new
+
+    @need.status = 'complete' if @type == 'log'
+    @need.user = current_user if @type == 'log'
+  end
+
+  def create
```

Comment:
Do we need to `authorize()` anything here?

Reply:
Ah, yes.  Good point